hack/verify_clientset.sh

```diff
@@ -35,12 +35,10 @@ mkdir -p "${TMP_DIFFROOT}"
 cp -a "${DIFFROOT}"/* "${TMP_DIFFROOT}"
 
 make clientset
-find "${TMP_DIFFROOT}" -name *.bazel -delete
 
 echo "diffing ${DIFFROOT} against freshly generated codegen"
 ret=0
 diff -Naupr "${DIFFROOT}" "${TMP_DIFFROOT}" -x '*.bazel' -x 'BUILD' || ret=$?
-cp -a "${TMP_DIFFROOT}"/* "${DIFFROOT}"
```

Comment:
@detiber i think this is line that causing issue you are seeing (correct me if not the same issue),
`make gazelle` will generate bazel under the `_tmp`, then it copy files from `_tmp` to main repo, 
then main repo got the bazel files which point to `_tmp` which is wrong.

what i found is: we don't need to copy those clientset files from `_tmp` to main repo. 
(i guess originally been done this way because: `make gazelle is not called after clientset been generated`, but thats only my guess)